www/src/views/starter-library/starter-list.js

```diff
@@ -102,7 +102,7 @@ const StartersList = ({ urlState, starters, count, sortRecent }) => {
                     >
                       {owner} /
                     </span>
-                    <span css={{ display: `flex` }}>
+                    <span css={{ display: `flex`, maxHeight: '1rem' }}>
```

Comment:
We should remove this one as it's more a hack than a fix.
```suggestion
                    <span css={{ display: `flex` }}>
```

Reply:
Thank you for taking the time to review and suggest edits for my commit. Yes your proposed changes is definitely less hacky.